CPS-0009/README.md

```diff
@@ -0,0 +1,104 @@
+---
+CPS: 9
+Title: Coin Selection Including Native Tokens
+Status: Open
+Category: Wallets
+Authors:
+  - Hinson Wong <wongkahinhinson@gmail.com>
+  - Tsz Wai Wu <wu.tszwai@outlook.com>
+Proposed Solutions: []
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/issues/232
+  - https://github.com/cardano-foundation/CIPs/pull/611
+Created: 2023-10-20
+---
+
+## Abstract
+
+The introduction of native tokens in Cardano has introduced unique challenges related to coin selection and transaction efficiency. This Cardano Problem Statement (CPS) addresses the need for a specialized coin selection approach to optimize transactions involving native tokens while minimizing transaction size and complexity.
+
+## Problem
+
+The integration of native tokens in Cardano transactions has brought both opportunities and challenges. One of the significant challenges is associated with coin selection when dealing with native tokens.
+
+### 1. Need for Efficient Coin Selection
+
+Cardano transactions involving native tokens often require attaching a minUTxO value (lovelace) to the transaction. In scenarios where multiple tokens are associated with a single UTxO, selecting such UTxOs as inputs can lead to inefficient transactions. This inefficiency arises from the increased transaction size due to token information, potentially impacting the decentralized applications and network performance.
+
+### 2. Prioritizing Necessary Tokens
```

Comment:
I don't understand the difference between problem 1 and 2.

For me it seems that problem 2 is a solution to problem 1, as by prioritising necessary tokens we make transactions more efficient.

Reply:
I've removed the problem 2, it does seem redundant after reading your comment.